src/pages/ProfilesExplorerView/components/SceneAiPanel/components/AiButton/AIButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { css } from '@emotion/css';
+import { IconName } from '@grafana/data';
+import { Button, useStyles2 } from '@grafana/ui';
+import { reportInteraction } from '@shared/domain/reportInteraction';
+import React, { ReactNode } from 'react';
+
+import { useFetchLlmPluginStatus } from './infrastructure/useFetchLlmPluginStatus';
+
+type AIButtonProps = {
+  children: ReactNode;
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  disabled?: boolean;
+  interactionName: string;
+};
+
+export function AIButton({ children, onClick, disabled, interactionName }: AIButtonProps) {
+  const styles = useStyles2(getStyles);
+  const { isEnabled, error, isFetching } = useFetchLlmPluginStatus();
+
+  let icon: IconName = 'ai';
+  let title = '';
+
+  if (error) {
+    icon = 'shield-exclamation';
+    title = 'Grafana LLM plugin missing or not configured!';
+  } else if (isFetching) {
+    icon = 'fa fa-spinner';
+    title = 'Checking the status of the Grafana LLM plugin...';
+  }
+
+  return (
+    <Button
```

Comment:
I took the opportunity to remove the "new" badge:

<img width="225" alt="image" src="https://github.com/user-attachments/assets/5a656634-e7ea-4d03-9bf7-304285594f6d">

The feature has already a few months now.